internal/engine/docker_compose_build_and_deployer.go

```diff
@@ -29,12 +28,11 @@ func (bd *DockerComposeBuildAndDeployer) BuildAndDeploy(ctx context.Context, man
 		return store.BuildResult{}, RedirectToNextBuilderf("not a docker compose manifest")
 	}
 
-	cmd := exec.CommandContext(ctx, "docker-compose", "-f", manifest.DCConfigPath, "up", "--no-deps", "--build", "--force-recreate", "-d", manifest.Name.String())
-	cmd.Stdout = logger.Get(ctx).Writer(logger.InfoLvl)
-	cmd.Stderr = logger.Get(ctx).Writer(logger.InfoLvl)
+	dcc := dockercompose.NewDockerComposeClient()
+	stdout := logger.Get(ctx).Writer(logger.InfoLvl)
```

Comment:
consider having the `cmdDCClient` just always write to the logger, and so when you call `NewD...Client()` it makes the logger and stores it on itself -- then you don't need to pass stdout/stderr in with every function call.

Reply:
...tbh we could even make a dockercompose client specific to the dc-yaml and then we wouldnt have to pass that in on every call, either, just at construction?

Reply:
yup agreed, I left a TODO for that